MEV_spec_RPC_v0_2.md

```diff
@@ -0,0 +1,141 @@
+---
+tags: spec
+---
+
+# MEV-Geth RPC v0.1
+
+# eth_sendBundle
+
+### Description
+
+Sends a bundle of transactions to the miner. The bundle has to be executed at the beginning of the block (before any other transactions), with bundle transactions executed exactly in the same order as provided in the bundle. During the Flashbots Alpha this is only called by the MEV-relay.
+
+| Name | Type | Description | Comment
+--------|----------|-----------|-----------
+transactions |	`Array<Data>` | Array of signed transactions (`eth_sendRawTransaction` style, signed and RLP-encoded)	| a no-op in the light mode
+blockNumber	|`Quantity`	|Exact block number at which the bundle can be included.	|bundle is evicted after the block
+minTimestamp	|`Quantity`	|Minimum (inclusive) block timestamp at which the bundle can be included. If this value is 0 then any timestamp is acceptable.
+maxTimestamp	|`Quantity`	|Maximum (inclusive) block timestamp at which the bundle can be included. If this value is 0 then any timestamp is acceptable.
+revertingTxHashes	|Array<`Data`>	|Array of tx hashes within the bundle that are allowed to cause the EVM execution to revert without preventing the bundle inclusion in a block.
```

Comment:
These data types (defined below) are different from those in the mev-geth spec doc.. are there strong reasons for this?